common/static/js/spec/main_requirejs.js

```diff
@@ -0,0 +1,161 @@
+(function(requirejs, define) {
+    requirejs.config({
+        paths: {
+            'gettext': 'js/test/i18n',
```

Comment:
Note to reviewer: I tried to be fairly conservative about which vendor files I included-- I stuck with things that I knew we needed, plus all the query, backbone, and jasmine variations.